packages/material-ui-system/src/index.d.ts

```diff
@@ -262,6 +263,417 @@ export const visuallyHidden: React.CSSProperties;
 export function unstable_getThemeValue(prop: string, value: any, theme: object): any;
+
+export type StandardCSSProperties = CSS.PropertiesFallback<number | string>;
+
+/**
+ * The `css` function accepts arrays as values for mobile-first responsive styles.
+ * Note that this extends to non-theme values also. For example `display=['none', 'block']`
+ * will also works.
+ *
+ * For more information see: https://styled-system.com/responsive-styles
+ */
+export type ResponsiveStyleValue<T> = T | Array<T | null> | { [key: string]: T | null };
+
+/**
+ * All non-vendor-prefixed CSS properties. (Allow `number` to support CSS-in-JS libs,
+ * since they are converted to pixels)
+ */
+export interface CSSProperties
+  extends CSS.StandardProperties<number | string>,
+    CSS.SvgProperties<number | string> {}
+
+/**
+ * Map of all CSS pseudo selectors (`:hover`, `:focus`, ...)
+ */
+export type CSSPseudoSelectorProps = { [K in CSS.Pseudos]?: SystemStyleObject };
+
+/**
+ * CSS as a plain object that is compatible with CSS-in-JS libaries.
+ * Copied directly from [emotion](https://github.com/emotion-js/emotion/blob/ca3ad1c1dcabf78a95b55cc2dc94cad1998a3196/packages/serialize/types/index.d.ts#L45) types
+ */
+export interface CSSObject
+  extends CSSPropertiesWithMultiValues,
+    CSSPseudosForCSSObject,
+    CSSOthersObjectForCSSObject {}
+
+export type CSSPropertiesWithMultiValues = {
+  [K in keyof CSSProperties]: CSSProperties[K];
+};
+export type CSSPseudosForCSSObject = { [K in CSS.Pseudos]?: CSSObject };
+export type CSSInterpolation = undefined | number | string | CSSObject;
+export interface CSSOthersObjectForCSSObject {
+  [propertiesName: string]: CSSInterpolation;
+}
+
+/**
+ * Map all nested selectors
+ */
+export interface CSSSelectorObject {
+  [cssSelector: string]: SystemStyleObject;
+}
+
+export interface AliasesCSSProperties {
+  /**
+   * The **`background-color`** CSS property sets the background color of an element.
+   *
+   * **Initial value**: `transparent`
+   *
+   * | Chrome | Firefox | Safari |  Edge  |  IE   |
+   * | :----: | :-----: | :----: | :----: | :---: |
+   * | **1**  |  **1**  | **1**  | **12** | **4** |
+   *
+   * @see https://developer.mozilla.org/docs/Web/CSS/background-color
+   */
+  bgcolor?: StandardCSSProperties['backgroundColor'];
+  /**
+   * The **`margin`** CSS property sets the margin area on all four sides of an element. It is a shorthand for `margin-top`, `margin-right`, `margin-bottom`, and `margin-left`.
+   *
+   * | Chrome | Firefox | Safari |  Edge  |  IE   |
+   * | :----: | :-----: | :----: | :----: | :---: |
+   * | **1**  |  **1**  | **1**  | **12** | **3** |
+   *
+   * @see https://developer.mozilla.org/docs/Web/CSS/margin
+   */
+  m?: StandardCSSProperties['margin'];
+  /**
+   * The **`margin-top`** CSS property sets the margin area on the top of an element. A positive value places it farther from its neighbors, while a negative value places it closer.
```

Comment:
```suggestion
   * The **`margin-top`** CSS property sets the margin on the top of an element. A positive value places it farther from its neighbors, while a negative value places it closer.
```

Reply:
Should this be `mt`?

Reply:
They are just aliases I believe that is why the description is using the CSS property name.. We may rename and use the system names everywhere